configs/encointer-kusama.yml

```diff
@@ -0,0 +1,25 @@
+# using chopsticks with encointer will not allow to perform actions involving encointer custom types which are
+# not supported by chopsticks: CommunityIdentifier, fixpoint balances
+# also, encointer uses custom rpc methods which are not available on chopsticks
+# still, basic chain functionality can be tested
```

Comment:
FYI.

Reply:
Chopsticks implements only the basic PRCs. Your custom RPCs methods are part of the node. Chopsticks only uses your runtime. Regarding types, you can define custom types on your config file